README.md

```diff
@@ -1,3 +1,34 @@
 # xk6-webcrypto
 
-This is a **work in progress** project implementation of the [WebCrypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) for k6.
\ No newline at end of file
+This is a **work in progress** project implementation of the [WebCrypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) specification for k6.
+
+## Current state
+
+The current state of the project is that it is an experimental, and incomplete implementation of the WebCrypto API specification. While we consider it ready for production use, it is still missing some features and is not yet fully compliant with the specification.
+
+### Supported APIs and algorithms
+
+#### Crypto
+
+- `crypto.getRandomValues()`
+- `crypto.randomUUID()`
+
+#### SubtleCrypto
+
+- `crypto.subtle.digest()` supports the complete specification's algorithms list: SHA-1, SHA-256, SHA-384, SHA-512.
+- `crypto.subtle.generateKey()` supports the following symmetric algorithms: AES-CBC, AES-GCM, and AES-CTR.
```

Comment:
I think a feature table could be more readable.

Reply:
You're right 👍🏻